src/plugins/dashboard/public/application/listing/dashboard_listing.test.tsx

```diff
@@ -166,7 +172,8 @@ describe('after fetch', () => {
 
   test('renders warning when listingLimit is exceeded', async () => {
     const services = makeDefaultServices();
-    services.savedObjects.settings.getListingLimit = () => 1;
+    pluginServices.getServices().dashboardCapabilities.showWriteControls = true;
+    pluginServices.getServices().settings.uiSettings.get = jest.fn().mockReturnValueOnce(1);
```

Comment:
This change caused the addition in the related snapshot, hence the snapshot update - however, this test doesn't seem to be even testing what it says it should be testing... So not even sure if it's worth keeping around?